File: Selenium/Project/LMS_Activity1.java
package project;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class LMS_Activity1 {
    public static void main(String[] args) {
     
        System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");

      
        WebDriver driver = new ChromeDriver();

        try {
           
            driver.get("https://alchemy.hguy.co/lms");

          
            String title = driver.getTitle();

           
            if ("Alchemy LMS – An LMS Application".equals(title)) {
                System.out.println("Title matches successfully.");
            } else {
                System.out.println("Title does not match. The actual title is: " + title);
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
          
            driver.quit();
        }
    }
}

File: Selenium/Project/lms_Activity1.py
# Goal: Read the title of the website and verify the text

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")
    driver.maximize_window()
        
    # Get  the title of the page and verify it
    page_title = driver.title.replace("–", "")
    print(page_title) 

    if(page_title=="Alchemy LMS  An LMS Application"):
        print("The Page title is : ",driver.title)
    else:
         print("Wrong Website ") 

    #close the browser
    driver.quit()